### PR TITLE
Update DocFX and use a different template

### PR DESCRIPTION
**Pull request by inftord**
yes.

### CODE REVIEW

File: docs/docfx.json [hunk 1]
```diff
@@ -1,4 +1,26 @@
 {
+  "metadata": [
+    {
+      "src": [
+        {
+	  "src": "../",
+          "files": [
+	    "src/OoLunar.DSharpPlus.CommandAll.csproj"
```
Comment (OoLunar) [inline docs/docfx.json]: The file in general isn't formatted

File: docs/docfx.json [hunk 2]
```diff
@@ -16,63 +38,25 @@
...
+    "output": "_site",
+    "globalMetadataFiles": [],
+    "fileMetadataFiles": [],
+    "template": [
+      "default",
+      "modern"
+    ],
+    "postProcessors": [],
+    "keepFileLink": false,
+    "disableGitFeatures": false
```
Comment (OoLunar) [inline docs/docfx.json]: How many properties here contain their default values?

**Review by OoLunar — CHANGES_REQUESTED** [review]
May I please have before/after screenshots before I merge? Additionally I see that you've set the `LangVersion` to preview. I assume this is for a docfx workaround since without it, docfx fails to build. Besides the below comments, LGTM.

File: docs/docfx.json (hunk 2 above)
Comment (inftord) [reply docs/docfx.json]: all of them, except xref. that's set so that links to d#+ entities work right

File: docs/docfx.json (hunk 1 above)
Comment (inftord) [reply docs/docfx.json]: i had to do all work in nano, can you perhaps format it?

Comment (OoLunar) [reply docs/docfx.json]: Yeah I can do it.

File: docs/docfx.json (hunk 2 above)
Comment (OoLunar) [reply docs/docfx.json]: So in theory, by removing all properties except `xref`, the docfx output will have no difference?

Comment (inftord) [reply docs/docfx.json]: .....don't try it, it could just break entirely

Comment (OoLunar) [reply docs/docfx.json]: I'm gonna try it

**Review by OoLunar — APPROVED** [review]

**Pull request merged**